src/patternfly/components/Tabs/tabs.scss

```diff
@@ -233,6 +260,10 @@ $pf-c-tabs--spacer-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl");
       --pf-c-tabs__link--before--BorderRightColor: var(--pf-c-tabs__item--m-current__link--Background);
       --pf-c-tabs__link--before--BorderBottomColor: var(--pf-c-tabs__link--before--BorderColor);
```

Comment:
should these vars that are being assigned have `m-current` in their name?

Reply:
`--pf-c-tabs__link--before--BorderRightColor: var(--pf-c-tabs__item--m-current__link--Background);` is just to hide the right border. `--pf-c-tabs__link--before--BorderBottomWidth` and `--pf-c-tabs__link--before--BorderBottomColor` shifts the border from side to bottom